js/ui/components/polkitAgent.js

```diff
@@ -150,6 +153,16 @@ const AuthenticationDialog = new Lang.Class({
         this._passwordBox.add(this._passwordEntry,
                               { expand: true });
 
+        this._inputSourceManager = InputSourceManager.getInputSourceManager();
+        this._inputSourceIndicator = new InputSourceIndicator(this, false);
+        this._passwordBox.add(this._inputSourceIndicator.container);
```

Comment:
Isn't the container already added to another actor, in the indicator code itself? Any reason not to use `this._inputSourceIndicator.actor` instead?

Reply:
The actor has a parent already:

Attempting to add actor of type 'ShellGenericContainer' to a container of type 'StBoxLayout', but the actor has already a parent of type 'StBin'.